apps/api-gateway/src/board/board.controller.ts

```diff
@@ -1,7 +1,55 @@
-import { Controller } from '@nestjs/common';
+import { Controller, Get, Post, Delete, Param, Body, Patch } from '@nestjs/common';
 import { BoardService } from './board.service';
+import { ApiCreatedResponse, ApiOperation } from '@nestjs/swagger';
+import { CreateBoardDto } from './dto/create-board.dto';
+import { UpdateBoardDto } from './dto/update-board.dto';
 
 @Controller('board')
 export class BoardController {
   public constructor(private readonly boardService: BoardService) {}
+
+  @Get()
+  @ApiOperation({ summary: '전체 게시글 조회' })
+  @ApiCreatedResponse({
+    description: '전체 게시글 조회',
+  })
+  public async getAllBoards() {
+    return await this.boardService.getAllBoards();
+  }
```

Comment:
```suggestion
  @Get()
  @ApiOperation({ summary: '전체 게시글 조회' })
  @ApiResponse(
  { 
    status : 200, 
    description : '전체 게시글 조회 성공',
    example : [
    {
        post_id : 1,
        title : '게시물 제목',
        content : '게시물 내용',
        update_at : 2024-10-01T00:00:12,
        create_at : 2024-10-03T00:00:12,
        expired_at: 2024-10-06T00:00:12 ,
        view_cnt : 15,
        bookmarked_cnt : 3,
        is_open : true
        },
        {
        post_id : 2,
        title : '게시물 제목2',
        content : '게시물 내용2',
        update_at : 2024-10-01T00:00:12,
        create_at : 2024-10-03T00:00:12,
        expired_at: 2024-10-06T00:00:12 ,
        view_cnt : 13,
        bookmarked_cnt : 52,
        is_open : true
        }
    ]
    })
  public async getAllBoards() {
    return await this.boardService.getAllBoards();
  }
```

이런 식의 ApiResponse에 대한 예시 데이터를 생성해보는 건 어떨까요!

Reply:
@ApiCreatedResponse 데코레이터의 type attribute를 이용해서 만들어줘도 괜찮은 것 같네요

Reply:
해당 부분 작성했습니다 감사합니다!
수정된 커밋입니다 : cd01ce5c3e8cefaf27805b94cee87ed8aacdf7c4